### PR TITLE
Cacheable: grammatical cleanup & clarity

### PR DESCRIPTION
**Pull request by t-dub**
### Description

I have made several purely grammatical fixes to the [Cacheable glossary entry](https://developer.mozilla.org/en-US/docs/Glossary/Cacheable) such as removing comma splices and correcting pluralization.  I've also attempted to improve the clarity of various parts of the explanation, removing some unnecessary embellishments to make the statements more authoritative and attempting to be more precise in talking about requests and responses.

### Motivation

Technical concepts are very challenge to explain clearly. I am hopeful that my revision helps.

### CODE REVIEW

File: files/en-us/glossary/cacheable/index.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ page-type: glossary-definition
...
-- There are _specific headers_ in the response, like {{HTTPHeader("Cache-Control")}}, that prevents caching.
+- The method used in the request is _cacheable_, that is either a {{HTTPMethod("GET")}} or a {{HTTPMethod("HEAD")}} method. A response to a {{HTTPMethod("POST")}} or {{HTTPMethod("PATCH")}} request can also be cached if freshness is indicated and the {{HTTPHeader("Content-Location")}} header is set, but this is rarely implemented. For example, Firefox does not support it ([Firefox bug 109553](https://bugzil.la/109553)). Other methods, like {{HTTPMethod("PUT")}} or {{HTTPMethod("DELETE")}} are not cacheable and their result cannot be cached.
+- The status code of the response is _known_ by the application caching, and is _cacheable_. The following status codes are cacheable: {{HTTPStatus("200")}}, {{HTTPStatus("203")}}, {{HTTPStatus("204")}}, {{HTTPStatus("206")}}, {{HTTPStatus("300")}}, {{HTTPStatus("301")}}, {{HTTPStatus("404")}}, {{HTTPStatus("405")}}, {{HTTPStatus("410")}}, {{HTTPStatus("414")}}, and {{HTTPStatus("501")}}.
+- There are not specific headers in the response, like {{HTTPHeader("Cache-Control")}}, with values that would prohibit caching.
```
Comment (dawei-wang) [inline files/en-us/glossary/cacheable/index.md:13]:
```suggestion
- There are no specific headers in the response, like {{HTTPHeader("Cache-Control")}}, with values that would prohibit caching.
```

**Review by dawei-wang — COMMENTED** [review]
typo

File: files/en-us/glossary/cacheable/index.md (hunk 1 above)
Comment (t-dub) [reply files/en-us/glossary/cacheable/index.md:13]: Heh, I actually _did_ mean "not" here, but I agree that "no" is actually better.

**Review by Josh-Cena — APPROVED** [review]
LGTM. Thanks for the cleanup!

**Pull request merged**